src/Core/src/Hosting/MauiHostEnvironment.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Maui.ApplicationModel;
+
+namespace Microsoft.Maui.Hosting;
+
+public class MauiHostEnvironment : IHostEnvironment
+{
+	public string EnvironmentName
+	{
+		get => "Production";
+		set => throw new System.NotImplementedException();
+	}
+
+	public string ApplicationName
+	{
+		get => AppInfo.Current.Name;
+		set => throw new System.NotImplementedException();
+	}
+
+	public string ContentRootPath
+	{ 
+		get => throw new System.NotImplementedException();
+		set => throw new System.NotImplementedException();
+	}
+
+	public IFileProvider ContentRootFileProvider
+	{ 
+		get => throw new System.NotImplementedException();
```

Comment:
@Eilon @javiercn @mkArtakMSFT would there be some benefit to attempting to use this implementation for the Blazor file provider? Is this the same concept here?